gulp/modules/Utilities.js

```diff
@@ -0,0 +1,148 @@
+var fs = require('fs');
+var Config = require('./Config');
+var path = require('path');
+
+/**
+ * Helpful assortment of Utilites used throughout the build
+ */
+var Utilities = function() {
+	/**
+     * Retrieves all months
+     * @returns {array} An array with the month names as strings
+     */
+	this.getMonths = function() {
```

Comment:
Nevermind, spacing looks fine in my editor. I think it might be rendering this way because tabs are being used instead of spaces.